doc/api/fs.markdown

```diff
@@ -363,6 +363,13 @@ descriptor.
 
 Change file timestamps of the file referenced by the supplied path.
 
+Note: the arguments `atime` and `mtime` of the following related functions does follow the
```

Comment:
I guess the lander could help me with fixing my grammar errors when landing my commits :(